test/unit/media-container.spec.js

```diff
@@ -0,0 +1,51 @@
+import { spy } from 'sinon';
+import { fixture, assert, aTimeout } from '@open-wc/testing';
+import '../../src/js/index.js';
+
+describe('<media-container>', () => {
```

Comment:
thought(non-blocking): We should probably either test the `<media-container>` directly or move these tests to the `<media-controller>` spec, since it is built on the assumption that `<media-controller/>` isn't overriding any of these behaviors. I'd be fine with either, and I don't think this should be treated as a blocker if you'd like to discuss further.